cmd/datamon/cmd/bundle.go

```diff
@@ -69,3 +69,12 @@ func setLatestOrLabelledBundle(ctx context.Context, remote context2.Stores) erro
 	log.Printf("Using bundle: %s", datamonFlags.bundle.ID)
 	return nil
 }
+
+func getConcurrencyFactor(batchSize int) int {
```

Comment:
✨ yes, totally forgot to add this coercion to prevent show-stoppers, which this could've been otherwise

Reply:
since concurrency is mostly driven by network consideration (bandwidth, latency), and not really driven by the capabilities of the local computer the program is running on, i don't think we can think of ever producing some automated factor, unless the execution is preceded by a brief benchmarking of gcs from the local machine.

Reply:
follow-up note: comment only. No change requested